src/rust/ensogl/lib/core/src/display/scene.rs

```diff
@@ -749,26 +762,48 @@ impl Views {
 /// FRP Scene interface.
 #[derive(Clone,CloneRef,Debug)]
 pub struct Frp {
-    pub network        : frp::Network,
-    pub shape          : frp::Sampler<Shape>,
-    pub camera_changed : frp::Stream,
-    pub frame_time     : frp::Stream<f32>,
-
-    camera_changed_source : frp::Source,
-    frame_time_source     : frp::Source<f32>,
+    pub network                    : frp::Network,
+    pub shape                      : frp::Sampler<Shape>,
+    pub camera_changed             : frp::Stream,
+    pub frame_time                 : frp::Stream<f32>,
+    pub processed_event            : frp::Stream<Option<web_sys::Event>>,
+    pub pass_processed_event_to_js : frp::Source<()>,
+    processed_event_source         : frp::Source<Option<web_sys::Event>>,
```

Comment:
this needs either documentation or renaming. In contrast to other FRP names, this name is not obvious to me. The name of course makes sense if you know what it does, but it doesnt if you dont.